scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -356,7 +356,7 @@ class ScanCode(
         }
 
         // Handling for ScanCode 2.9.7 and below.
-        return result["files_count"].intValue()
+        return result["files_count"]?.intValue() ?: 0
```

Comment:
Now the comment on line 358 isn't true anymore because line 359 is also for scancode post 3.9.7 in case there is no result. Anyhow instead of requiring each `get<SomethingFrom>(result: Json)` to handle `no result was generated` case it seems simpler to handle the error as far as possible upwards in the calltree. What do you think about that ?

Reply:
> it seems simpler to handle the error as far as possible upwards in the calltree

I was considering that, but as `getLicenseFindings()`, `getCopyrightFindings()` and `getErrors()` are already safe in that regard, it seemed more consistent to me to make the last missing piece (i.e. `getFileCount()`) also safe.

Reply:
Makes sense to me as well, thanks.